content/en/service-mesh.md

```diff
@@ -26,3 +26,21 @@ Service meshes add reliability, observability, and security features
 uniformly across all services across a cluster without requiring code changes. 
 Before service meshes, that functionality had to be encoded into every single service, 
 becoming a potential source of bugs and technical debt.
+
+In the [Sidecar Container](/sidecar-container/) model, 
+each microservice pod is paired with its own proxy. 
+This per-service proxy intercepts and manages network traffic, 
+enforce security policies, balance workloads, 
+and collect performance data for each individual service. 
+While this approach offers excellent control and service-specific traffic management,
+it also uses more computing resources and becomes more complex to manage as your system grows.
+While this approach offers maximal policy flexibility and service-specific routing,
+it also incurs additional CPU and memory overhead for every proxy and increases operational complexity as the mesh grows.
```

Comment:
I'd suggest merging these two sentences, they start of identically and are repeating a similar idea.

Reply:
Line +37 to +38 shouldn't be there! I'll add the suggestion to remove them.